mangaki/mangaki/utils/work_merge.py

```diff
@@ -91,14 +91,20 @@ def field_changeset(works):
         for field in work:
             rows[field].append(work[field])
 
+    strip_protocol = re.compile(r'https?://')
     for field, choices in rows.items():
+        difficulty = 0
         suggested = None
 
         if field == 'id':
             action = ActionType.JUST_CONFIRM
-            suggested = min(choices)
+            suggested = min(int(choice) for choice in choices)
```

Comment:
Nit: `min(map(int, choices))` is simpler but alright for using generator expressions.

Reply:
Simpler but less readable I think.

Reply:
Or should I say less comprehensible krkr.